RecoLocalTracker/SiPixelClusterizer/test/TestClusters.cc

```diff
@@ -2304,7 +2304,7 @@ void TestClusters::analyze(const edm::Event &e, const edm::EventSetup &es) {
   int numOfPixPerDisk3 = 0;
   int numOfPixPerDisk4 = 0;
 
-  float avCharge1 = 0., avCharge2 = 0., avCharge3 = 0., avCharge4 = 0., avCharge5 = 0.;
+  //float avCharge1 = 0., avCharge2 = 0., avCharge3 = 0., avCharge4 = 0., avCharge5 = 0.;
```

Comment:
Wouldn't be better to remove it?
(same comment for all the others commented lines)

Reply:
@clacaputo , I did not delete it as there is some commented out code ( e.g. see code after https://github.com/cms-sw/cmssw/blob/19451fc97b6dda130b29624d4540575f1e43f250/RecoLocalTracker/SiPixelClusterizer/test/TestClusters.cc#L3536 ) which uses these variables.